styles/styles.css

```diff
@@ -0,0 +1,534 @@
+  color: #ffffff;
+  text-decoration: none;
+}
+footer img {
+  width: 70%;
+  margin: 0 15% 0;
+}
+label {
+  font-family: 'Noto Serif', serif;
+  font-size: 2rem;
+  color: #093c60;
+}
+legend {
+  font-family: 'Noto Serif', serif;
+  font-size: 2rem;
+  color: #093c60;
+}
+input, select, textarea {
+  font-family: Arial, sans-serif;
+  font-size: 2rem;
+  color: #444;
+  box-sizing: border-box;
+  display: block;
+  margin: 1rem 0rem 1rem;
+  padding: 1rem;
+}
+input {
+  width: 100%;
+  border-radius: .5rem;
+  border: 1px
+  solid #aaa;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+input[type="submit"] {
+  width: 20rem;
+  background-color: #81b741;
+  color: #fff;
+}
+input:focus {
+  background-color: #093c60;
+  outline: none;
+}
+textarea {
+  color: #aaa;
+  border: 1px solid #aaa;
+  border-radius: .5rem;
+  width: 100%;
+}
+textarea:focus {
+  color: #444;
+  background-color: #ececec;
+  outline: none;
+}
+fieldset {
+  border: none;
+  padding: 0rem 0rem 3rem;
+}
+fieldset input {
+  display: inline;
+}
+.radio input {
+  position: relative;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+.radio label {
+  font-family: Arial, sans-serif;
+  font-size: 1rem;
+  color: #444;
+  display: block;
+  position: relative;
+  margin-top: 12px;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+.rodeo {
+  position: absolute;
+  top: 20px;
+  left: 0;
+  height: 25px;
+  width: 25px;
+  background-color: #ececec;
+  border-radius: 50%;
+}
+.radio:hover input ~ .rodeo {
+  background-color: #ececec;
+}
+.radio input:checked ~ .rodeo {
+  background-color: #81b741;
+}
+.rodeo:after {
+  content: "";
+  position: relative;
+  display: block;
+}
+.radio input:checked ~ .rodeo:after {
+  display: block;
+}
+.radio .rodeo:after {
+  top: 9px;
+  left: 9px;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  background: white;
+}
+.adopted-container {
+  display: flex;
+}
+.adopted-box {
+  width: 20rem;
+  margin: 0rem 2rem 0rem 0rem;
+}
+.adopted-box img {
+  width: 5rem;
+  height: 5rem;
+  margin: 0rem 1rem 0rem 0rem;
+  object-fit: cover;
+  float: left;
+}
+.adoptee {
+  margin: 0rem 0rem 0rem;
+  font-family: 'Noto Serif', serif;
+  color: #093c60;
+  font-size: 1.5rem;
+}
+.blog-container {
+  display: block;
+}
+.blog-intro {
+  font-size: 2rem;
+  font-weight: lighter;
+  line-height: 3rem;
+  color: #888;
+}
+.blog {
+  font-size: 2rem;
+  font-weight: lighter;
+  line-height: 3rem;
+  color: #444;
+}
+
+@media screen and (min-width: 992px) {
+* {
+    font-size: 1rem;
+  }
+header {
+    padding: 0rem 6rem 4rem;
+  }
+.logo-img {
+    width: 100%;
+  }
+.tagline-text {
+    font-size: 1.5rem;
+  }
+.head-box {
+    min-width: 200px;
+  }
+nav {
+    padding: 1rem 12rem 1rem;
+  }
+nav p {
+    font-size: 1.5rem;
+}
+nav p a {
+    font-size: 1.5rem;
+}
+article {
+    padding: 3rem 12rem 3rem;
```

Comment:
I would play around with these values. I found the page to look better when this was disabled.